lib/mcinstall/index.js

```diff
@@ -0,0 +1,61 @@
+import {BaseServicePlist} from '../base-service';
+import {fs, plist} from '@appium/support';
+
+
+const MC_INSTALL_SERVICE_NAME = 'com.apple.mobile.MCInstall';
+const ACKNOWLEDGED = 'Acknowledged';
+
+function checkACK (res, name) {
+  if (res.Status !== ACKNOWLEDGED) {
+    throw new Error(`${name} error: ${JSON.stringify(res)}`);
+  }
+  return res;
+}
+
+class MCInstallProxyService extends BaseServicePlist {
+  constructor (socketClient) {
+    super(socketClient);
+  }
+
+  /**
+   * Get all profiles of iOS devices
+   * @returns {Promise<Object>}
```

Comment:
Would it be possible to describe an example response from this API?

Reply:
I updated this describe